#! /bin/sh /usr/share/dpatch/dpatch-run
## 01_fix_warnings.dpatch by Roland Rosenfeld <roland@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Try to fix several compiler warnings by correcting printf()
## DP: parameters, adding and fixing declarations etc.

@DPATCH@
diff -urNad xfig~/u_free.c xfig/u_free.c
--- xfig~/u_free.c
+++ xfig/u_free.c
@@ -168,14 +168,14 @@
 	return;
 
     if (picture->refcount == 0) {
-	fprintf(stderr, "Error freeing picture %x %s with refcount = 0\n",
+	fprintf(stderr, "Error freeing picture %p %s with refcount = 0\n",
 		picture, picture->file);
 	return;
     }
     picture->refcount--;
     if (picture->refcount == 0) {
 	if (appres.DEBUG)
-	    fprintf(stderr,"Delete picture %x %s, refcount = %d\n",
+	    fprintf(stderr,"Delete picture %p %s, refcount = %d\n",
 				picture, picture->file, picture->refcount);
 	if (picture->bitmap)
 	    free((char *) picture->bitmap);
@@ -193,7 +193,7 @@
 	free(picture);
     } else {
 	if (appres.DEBUG)
-	    fprintf(stderr,"Decrease refcount for picture %x %s, refcount = %d\n",
+	    fprintf(stderr,"Decrease refcount for picture %p %s, refcount = %d\n",
 				picture, picture->file, picture->refcount);
     }
 }
diff -urNad xfig~/w_file.c xfig/w_file.c
--- xfig~/w_file.c
+++ xfig/w_file.c
@@ -208,7 +208,7 @@
     *iyoff = round(yoff*offset_unit_conv[yoff_unit_setting]);
 }
 
-void
+static void
 merge_request(Widget w, XButtonEvent *ev)
 {
     if (preview_in_progress) {
@@ -291,7 +291,7 @@
     }
 }
 
-void
+static void
 do_load(Widget w, XButtonEvent *ev)
 {
     char	    fname[PATH_MAX];
#! /bin/sh /usr/share/dpatch/dpatch-run
## 03_usr_share.dpatch by Roland Rosenfeld <roland@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Change documentation, that image libraries are in
## /usr/share/xfig and documentation is in /usr/share/doc/xfig.

@DPATCH@
diff -urNad xfig~/Doc/xfig.man xfig/Doc/xfig.man
--- xfig~/Doc/xfig.man
+++ xfig/Doc/xfig.man
@@ -1474,8 +1474,11 @@
 See the
 .IR Imakefile .
 .TP
-/usr/lib/X11/xfig
-This directory contains the html documentation, the above mentioned
+/usr/share/doc/xfig
+This directory contains the documentation.
+.TP
+/usr/share/xfig
+This directory contains the above mentioned
 CompKeyDB file, and a directory of libraries containing Fig objects
 such as electrical symbols, logic symbols, etc.
 .TP
diff -urNad xfig~/Doc/xfig_man.html xfig/Doc/xfig_man.html
--- xfig~/Doc/xfig_man.html
+++ xfig/Doc/xfig_man.html
@@ -2504,8 +2504,11 @@
 See the
 <I>Imakefile</I>.
 
-<DT>/usr/lib/X11/xfig<DD>
-This directory contains the html documentation, the above mentioned
+<DT>/usr/share/doc/xfig<DD>
+This directory contains the documentation.
+
+<DT>/usr/share/xfig<DD>
+This directory contains the above mentioned
 CompKeyDB file, and a directory of libraries containing Fig objects
 such as electrical symbols, logic symbols, etc.
 <DT><B>Fig.ad and Fig-color.ad</B>
#! /bin/sh /usr/share/dpatch/dpatch-run
## 04_app-defaults.dpatch by Roland Rosenfeld <roland@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Adusts app-defaults file to Debian requirements.

@DPATCH@
#! /bin/sh /usr/share/dpatch/dpatch-run
## 05_xfig-libs.dpatch by Roland Rosenfeld <roland@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: mention xfig-libs package, if no libraries are available.

@DPATCH@
diff -urNad xfig~/w_library.c xfig/w_library.c
--- xfig~/w_library.c
+++ xfig/w_library.c
@@ -1367,7 +1367,7 @@
     path = appres.library_dir;
 
     if (stat(path, &st) != 0) {       /* no such file */
-	file_msg("Can't find %s, no libraries available", path);
+	file_msg("Can't find %s, no libraries available, please install package xfig-libs", path);
 	return 0;
     } else if (S_ISDIR(st.st_mode)) {
 	/* if it is directory, scan the sub-directories and search libraries */
#! /bin/sh /usr/share/dpatch/dpatch-run
## 06_manual_version.dpatch by Roland Rosenfeld <roland@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Add version and date to ALT-tag of the title logo in
## DP: introduction.html (Closes: #230578).

@DPATCH@
diff -urNad xfig~/Doc/html/introduction.html xfig/Doc/html/introduction.html
--- xfig~/Doc/html/introduction.html
+++ xfig/Doc/html/introduction.html
@@ -7,7 +7,7 @@
 </HEAD>
 <BODY>
 <a href="contents.html"><img src="images/xfig-title.png"
-  alt="XFIG Users Manual" border=0></a>
+  alt="XFIG Users Manual Version 3.2.5 Feb 22, 2007" border=0></a>
 
 <P>[ <img src="images/usflag.png" alt=""><B>English Version</B>
  | <A HREF="japanese/index.html" target="_top"><img src="images/japaneseflag.png" alt="" border=0>
#! /bin/sh /usr/share/dpatch/dpatch-run
## 07_maps_europe.dpatch by Roland Rosenfeld <roland@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Correct description of Maps/Europe/central_europe and add
## DP: description to eastern_europe (Closes: #197592).


@DPATCH@
diff -urNad xfig~/Libraries/Maps/Europe/eastern_europe.fig xfig/Libraries/Maps/Europe/eastern_europe.fig
--- xfig~/Libraries/Maps/Europe/eastern_europe.fig
+++ xfig/Libraries/Maps/Europe/eastern_europe.fig
@@ -6,6 +6,7 @@
 100.00
 Single
 -2
+# Eastern Europe
 1200 2
 # Estonia
 2 3 0 1 0 7 48 -1 -1 0.000 1 0 7 0 0 96
#! /bin/sh /usr/share/dpatch/dpatch-run
## 08_maps_span.dpatch by Eugen Dedu <Eugen.Dedu@pu-pm.univ-fcomte.fr>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Remove Portugal from Maps/Europe/spain.fig, otherwise this DP:
## should be named iberia.fig (Closes: #258224).

@DPATCH@
diff -urNad xfig~/Libraries/Maps/Europe/spain.fig xfig/Libraries/Maps/Europe/spain.fig
--- xfig~/Libraries/Maps/Europe/spain.fig
+++ xfig/Libraries/Maps/Europe/spain.fig
@@ -1,245 +1,265 @@
-#FIG 3.2
-Portrait
+#FIG 3.2  Produced by xfig version 3.2.5-alpha5
+Landscape
 Center
 Metric
 A4      
 100.00
 Single
 -2
-# Spain
 1200 2
 # Spain
-2 3 0 1 0 7 48 0 -1 0.000 0 0 7 0 0 1393
-	 3290 8388 3287 8388 3279 8388 3275 8388 3272 8388 3272 8391
-	 3268 8391 3264 8391 3260 8391 3257 8395 3253 8395 3238 8395
-	 3227 8399 3219 8399 3212 8403 3204 8403 3197 8403 3193 8403
-	 3189 8403 3182 8399 3174 8395 3167 8391 3167 8388 3159 8384
-	 3155 8380 3144 8376 3137 8373 3129 8373 3122 8369 3114 8369
-	 3107 8369 3095 8369 3084 8369 3077 8369 3069 8365 3069 8361
-	 3065 8361 3065 8354 3062 8346 3062 8339 3058 8331 3058 8324
-	 3058 8316 3054 8313 3054 8309 3050 8309 3047 8301 3047 8298
-	 3047 8294 3043 8290 3035 8286 3032 8286 3028 8283 3024 8279
-	 3020 8275 3017 8275 3013 8271 3005 8268 2998 8264 2979 8256
-	 2972 8253 2968 8249 2960 8245 2957 8241 2953 8238 2949 8234
-	 2942 8230 2942 8226 2934 8223 2927 8219 2919 8215 2912 8211
-	 2904 8208 2897 8204 2893 8196 2889 8193 2885 8189 2878 8185
-	 2874 8181 2867 8178 2859 8174 2859 8178 2852 8178 2844 8181
-	 2833 8185 2825 8193 2822 8196 2818 8204 2814 8208 2810 8211
-	 2807 8211 2803 8208 2799 8204 2795 8204 2792 8200 2788 8193
-	 2784 8193 2777 8189 2769 8189 2762 8189 2750 8189 2743 8185
-	 2735 8185 2728 8181 2709 8178 2702 8178 2690 8174 2690 8170
-	 2683 8166 2679 8163 2672 8159 2668 8151 2664 8148 2664 8140
-	 2660 8140 2660 8136 2657 8133 2649 8125 2642 8121 2638 8118
-	 2634 8118 2627 8114 2615 8110 2612 8110 2608 8110 2597 8106
-	 2585 8106 2582 8106 2578 8106 2574 8103 2574 8099 2574 8095
-	 2574 8091 2570 8088 2570 8084 2567 8080 2563 8073 2559 8069
-	 2556 8069 2552 8065 2544 8065 2541 8065 2537 8061 2529 8054
-	 2526 8050 2522 8046 2518 8043 2511 8039 2507 8035 2507 8031
-	 2499 8028 2492 8024 2492 8020 2484 8020 2477 8016 2469 8013
-	 2466 8009 2462 8005 2454 8001 2451 8001 2447 7998 2443 7998
-	 2439 7998 2436 7998 2432 7998 2424 7998 2421 7994 2424 7990
-	 2424 7986 2428 7986 2428 7983 2432 7979 2436 7975 2443 7971
-	 2447 7964 2454 7960 2454 7956 2454 7953 2454 7949 2451 7945
-	 2451 7941 2443 7938 2439 7934 2432 7930 2428 7930 2421 7930
-	 2417 7926 2413 7926 2409 7926 2406 7923 2402 7919 2398 7915
-	 2394 7915 2391 7908 2391 7904 2391 7900 2370 7890 2364 7885
-	 2353 7881 2349 7881 2342 7878 2334 7874 2327 7874 2319 7870
-	 2312 7870 2304 7870 2301 7870 2297 7870 2297 7866 2293 7866
-	 2289 7866 2286 7866 2278 7866 2278 7863 2274 7863 2267 7859
-	 2263 7859 2259 7855 2256 7851 2252 7848 2248 7844 2244 7840
-	 2241 7836 2237 7829 2229 7825 2226 7821 2218 7818 2211 7810
-	 2203 7806 2199 7806 2192 7803 2184 7799 2173 7799 2169 7799
-	 2162 7799 2154 7799 2151 7803 2143 7803 2132 7803 2124 7803
-	 2121 7799 2113 7795 2106 7795 2098 7791 2098 7788 2091 7784
-	 2083 7784 2076 7780 2072 7776 2061 7773 2061 7769 2057 7765
-	 2057 7758 2057 7750 2057 7743 2057 7735 2057 7731 2057 7728
-	 2057 7720 2057 7716 2053 7713 2042 7701 2034 7698 2027 7698
-	 2019 7694 2008 7694 2000 7694 1989 7698 1982 7698 1970 7698
-	 1963 7701 1955 7701 1948 7701 1937 7701 1925 7701 1910 7701
-	 1899 7701 1888 7701 1877 7701 1865 7698 1850 7694 1843 7694
-	 1832 7686 1820 7683 1813 7679 1805 7675 1802 7671 1794 7668
-	 1783 7660 1772 7653 1764 7645 1753 7641 1745 7634 1738 7630
-	 1734 7630 1727 7626 1715 7626 1715 7623 1708 7619 1704 7615
-	 1700 7611 1693 7611 1693 7608 1685 7604 1682 7600 1678 7596
-	 1674 7593 1670 7589 1667 7585 1667 7578 1663 7566 1663 7559
-	 1663 7551 1663 7544 1659 7536 1655 7533 1655 7529 1648 7521
-	 1640 7521 1637 7521 1633 7518 1625 7514 1614 7510 1607 7506
-	 1603 7499 1595 7495 1588 7491 1580 7488 1573 7484 1562 7476
-	 1558 7469 1547 7465 1543 7465 1539 7465 1535 7469 1532 7469
-	 1528 7473 1528 7476 1524 7480 1524 7484 1520 7491 1517 7495
-	 1513 7499 1509 7506 1505 7510 1502 7510 1494 7514 1487 7514
-	 1479 7514 1472 7514 1460 7514 1449 7514 1445 7510 1438 7510
-	 1427 7506 1419 7503 1415 7499 1408 7495 1404 7491 1397 7488
-	 1389 7484 1382 7480 1374 7480 1367 7476 1363 7476 1356 7473
-	 1352 7469 1348 7465 1344 7461 1337 7458 1333 7454 1333 7450
-	 1326 7446 1322 7439 1318 7435 1311 7428 1303 7420 1299 7416
-	 1296 7409 1292 7409 1284 7401 1273 7401 1269 7401 1266 7401
-	 1262 7401 1258 7398 1254 7394 1251 7386 1247 7383 1239 7379
-	 1232 7375 1224 7371 1217 7368 1213 7368 1206 7364 1198 7364
-	 1191 7364 1187 7364 1172 7371 1164 7375 1161 7379 1157 7383
-	 1149 7386 1146 7394 1142 7401 1138 7405 1134 7413 1134 7416
-	 1131 7420 1131 7424 1127 7428 1123 7431 1116 7431 1116 7435
-	 1112 7439 1108 7439 1104 7439 1097 7443 1093 7446 1089 7446
-	 1086 7446 1078 7450 1071 7450 1063 7450 1059 7446 1052 7446
-	 1044 7443 1037 7439 1029 7435 1026 7431 1018 7428 1010 7424
-	 1007 7424 999 7420 992 7416 984 7416 977 7416 973 7416
-	 969 7413 965 7413 962 7413 958 7416 954 7416 947 7416
-	 943 7416 939 7420 935 7420 928 7420 920 7420 913 7420
-	 905 7420 898 7420 894 7420 890 7424 887 7424 883 7428
-	 883 7435 879 7439 879 7443 879 7446 875 7450 875 7454
-	 872 7454 872 7458 868 7458 868 7461 868 7465 868 7473
-	 868 7480 868 7488 872 7491 872 7499 875 7503 879 7510
-	 879 7514 879 7521 883 7529 883 7536 883 7540 883 7544
-	 883 7551 879 7555 879 7563 883 7570 887 7581 887 7585
-	 887 7589 887 7593 887 7596 883 7600 879 7604 875 7611
-	 879 7615 887 7623 894 7626 902 7630 909 7630 913 7634
-	 917 7638 917 7641 913 7645 913 7649 913 7641 909 7641
-	 905 7649 902 7656 894 7664 890 7675 887 7679 883 7686
-	 875 7701 872 7709 872 7713 872 7720 868 7728 868 7735
-	 868 7743 868 7746 864 7758 864 7765 860 7769 853 7780
-	 849 7791 842 7795 842 7803 842 7806 842 7814 842 7825
-	 842 7829 842 7836 842 7844 842 7851 842 7859 842 7866
-	 842 7874 842 7881 838 7889 838 7896 838 7904 838 7911
-	 834 7919 838 7926 838 7934 838 7941 838 7949 838 7956
-	 838 7964 834 7971 834 7979 830 7990 830 8005 830 8016
-	 830 8031 830 8039 830 8043 830 8050 830 8054 827 8061
-	 823 8069 819 8076 815 8084 812 8091 804 8103 801 8118
-	 793 8125 793 8129 789 8136 786 8140 782 8144 778 8148
-	 774 8155 767 8163 763 8166 759 8174 756 8178 748 8185
-	 744 8193 741 8200 733 8215 729 8223 722 8230 718 8238
-	 714 8241 711 8249 707 8253 699 8264 696 8275 688 8290
-	 681 8305 677 8313 669 8320 666 8328 662 8331 654 8339
-	 647 8350 639 8361 636 8365 628 8376 624 8388 617 8399
-	 606 8410 587 8425 583 8429 579 8433 579 8436 576 8440
-	 568 8448 564 8451 557 8459 553 8463 553 8466 549 8466
-	 546 8466 542 8466 538 8466 527 8466 523 8466 523 8470
-	 519 8474 516 8478 512 8481 508 8481 508 8485 504 8485
-	 501 8489 497 8496 493 8496 489 8500 489 8508 482 8519
-	 474 8526 471 8534 463 8541 463 8549 456 8556 452 8560
-	 448 8564 448 8568 441 8575 437 8583 437 8586 433 8590
-	 433 8594 433 8598 433 8601 429 8609 429 8613 426 8616
-	 426 8620 426 8624 422 8628 422 8631 418 8635 422 8639
-	 426 8643 429 8646 433 8646 437 8646 441 8646 452 8643
-	 459 8639 471 8631 482 8628 489 8624 493 8624 497 8620
-	 504 8620 512 8616 516 8616 519 8613 531 8613 534 8613
-	 538 8616 542 8620 546 8624 546 8631 546 8639 546 8646
-	 546 8658 546 8661 546 8669 546 8673 542 8673 542 8676
-	 538 8680 516 8703 508 8710 504 8710 504 8714 501 8718
-	 501 8721 501 8729 501 8733 501 8736 504 8740 508 8744
-	 516 8748 519 8751 523 8755 523 8763 527 8766 527 8774
-	 531 8785 531 8793 531 8800 531 8808 527 8819 527 8826
-	 523 8834 519 8845 516 8853 512 8860 512 8868 508 8871
-	 508 8883 508 8886 504 8894 504 8898 501 8901 501 8909
-	 501 8913 497 8916 497 8924 493 8931 493 8943 493 8950
-	 489 8958 489 8961 489 8969 486 8969 482 8976 478 8984
-	 471 8995 463 9003 456 9010 448 9021 441 9036 433 9044
-	 429 9051 426 9059 422 9066 422 9070 418 9078 411 9085
-	 407 9093 399 9096 396 9104 392 9108 384 9111 381 9119
-	 377 9123 377 9126 377 9134 384 9141 392 9145 399 9149
-	 407 9149 418 9153 429 9153 437 9153 444 9156 452 9156
-	 459 9156 467 9156 474 9160 486 9164 493 9168 501 9171
-	 508 9175 516 9183 519 9186 527 9194 531 9198 538 9201
-	 546 9209 553 9213 561 9216 568 9220 572 9224 576 9228
-	 587 9231 594 9235 598 9239 606 9243 613 9246 617 9246
-	 628 9250 632 9250 643 9246 651 9246 658 9243 666 9243
-	 673 9243 684 9239 692 9239 703 9243 711 9243 718 9243
-	 726 9243 733 9243 748 9246 756 9246 763 9250 771 9250
-	 774 9254 782 9258 786 9261 793 9261 793 9265 801 9269
-	 808 9276 819 9280 827 9288 834 9291 838 9295 845 9303
-	 853 9306 857 9314 860 9314 860 9318 864 9325 868 9329
-	 872 9333 872 9340 879 9348 879 9351 883 9355 883 9359
-	 887 9366 887 9378 890 9385 890 9396 890 9404 894 9411
-	 894 9419 898 9426 898 9430 898 9434 898 9438 898 9441
-	 898 9449 898 9456 898 9468 898 9471 894 9475 894 9483
-	 894 9490 890 9494 890 9501 890 9509 890 9516 890 9524
-	 898 9535 898 9543 898 9546 902 9554 913 9569 917 9569
-	 928 9576 932 9576 932 9584 932 9591 935 9599 939 9606
-	 943 9610 947 9614 947 9621 950 9625 954 9633 958 9636
-	 962 9644 965 9648 969 9651 980 9655 988 9663 995 9666
-	 1003 9670 1010 9674 1014 9678 1018 9681 1026 9685 1026 9689
-	 1029 9689 1041 9693 1048 9696 1056 9696 1067 9700 1074 9700
-	 1086 9700 1089 9696 1097 9696 1123 9670 1127 9666 1127 9663
-	 1131 9659 1138 9651 1142 9648 1146 9644 1149 9640 1153 9636
-	 1161 9633 1161 9629 1168 9625 1172 9621 1179 9621 1179 9618
-	 1183 9618 1187 9614 1191 9614 1194 9614 1198 9614 1202 9614
-	 1206 9614 1213 9618 1221 9618 1228 9621 1232 9621 1243 9621
-	 1251 9621 1254 9621 1258 9621 1262 9621 1266 9621 1273 9621
-	 1281 9621 1288 9621 1292 9618 1299 9618 1307 9614 1314 9614
-	 1322 9606 1329 9606 1333 9603 1337 9599 1341 9599 1344 9595
-	 1348 9595 1352 9595 1359 9595 1363 9595 1371 9595 1374 9595
-	 1382 9599 1389 9603 1397 9606 1404 9606 1408 9610 1415 9610
-	 1419 9614 1427 9618 1434 9618 1442 9621 1445 9621 1453 9621
-	 1460 9625 1468 9629 1475 9629 1483 9633 1490 9636 1498 9636
-	 1505 9636 1513 9640 1524 9640 1535 9644 1550 9648 1558 9648
-	 1569 9651 1573 9651 1577 9651 1580 9651 1588 9651 1595 9655
-	 1603 9655 1610 9655 1622 9655 1629 9659 1633 9659 1640 9659
-	 1644 9663 1648 9666 1652 9666 1655 9666 1659 9670 1667 9674
-	 1674 9678 1682 9681 1685 9685 1693 9689 1697 9693 1700 9696
-	 1704 9700 1708 9704 1712 9711 1715 9711 1723 9715 1727 9715
-	 1734 9719 1738 9719 1742 9719 1745 9715 1753 9711 1760 9708
-	 1764 9700 1768 9696 1775 9696 1779 9693 1783 9693 1787 9693
-	 1790 9693 1798 9693 1805 9693 1817 9696 1820 9696 1824 9700
-	 1828 9708 1832 9711 1835 9715 1835 9719 1843 9726 1850 9726
-	 1858 9730 1869 9734 1877 9734 1888 9734 1895 9734 1899 9730
-	 1899 9726 1903 9723 1903 9719 1907 9711 1910 9708 1922 9696
-	 1929 9693 1940 9685 1948 9678 1955 9670 1967 9663 1970 9655
-	 1978 9648 1982 9640 1982 9633 1985 9625 1989 9618 1989 9606
-	 1993 9603 1993 9595 1997 9588 2000 9584 2004 9580 2012 9576
-	 2019 9576 2027 9576 2030 9576 2038 9576 2049 9573 2053 9573
-	 2064 9573 2072 9569 2083 9569 2094 9565 2102 9565 2113 9565
-	 2121 9561 2128 9561 2136 9561 2143 9561 2151 9565 2158 9565
-	 2166 9569 2173 9569 2181 9569 2188 9569 2196 9565 2199 9565
-	 2207 9561 2211 9561 2218 9558 2222 9558 2226 9554 2229 9550
-	 2233 9543 2237 9535 2237 9531 2237 9524 2237 9520 2233 9513
-	 2233 9509 2233 9501 2237 9494 2241 9486 2244 9479 2248 9475
-	 2256 9468 2271 9441 2271 9434 2274 9430 2274 9426 2278 9419
-	 2282 9415 2286 9411 2289 9408 2293 9404 2297 9400 2301 9396
-	 2304 9393 2308 9389 2316 9385 2323 9381 2323 9378 2331 9374
-	 2334 9370 2338 9366 2346 9363 2353 9355 2361 9355 2368 9351
-	 2376 9348 2379 9344 2387 9340 2394 9340 2402 9336 2409 9336
-	 2417 9333 2424 9333 2432 9329 2436 9325 2443 9325 2451 9321
-	 2458 9321 2466 9318 2469 9318 2473 9314 2477 9310 2481 9310
-	 2484 9306 2488 9306 2496 9303 2499 9299 2503 9299 2507 9295
-	 2511 9291 2514 9291 2518 9288 2518 9284 2518 9280 2514 9273
-	 2511 9269 2507 9265 2503 9261 2499 9258 2496 9254 2492 9250
-	 2484 9246 2484 9243 2481 9235 2473 9231 2469 9216 2466 9213
-	 2462 9205 2458 9201 2458 9194 2451 9186 2447 9171 2443 9156
-	 2443 9153 2439 9145 2436 9138 2436 9126 2436 9119 2439 9078
-	 2443 9070 2443 9063 2443 9059 2447 9059 2447 9055 2443 9051
-	 2443 9048 2439 9048 2436 9044 2432 9040 2432 9036 2432 9029
-	 2432 9025 2436 9021 2436 9018 2439 9014 2443 9010 2447 9006
-	 2451 9003 2454 8999 2462 8991 2466 8988 2473 8984 2477 8976
-	 2481 8973 2484 8969 2488 8969 2492 8965 2496 8961 2503 8961
-	 2507 8954 2514 8950 2522 8946 2533 8943 2537 8939 2541 8935
-	 2544 8931 2548 8931 2552 8928 2559 8924 2563 8920 2567 8916
-	 2570 8916 2574 8913 2578 8909 2582 8905 2585 8901 2593 8898
-	 2597 8894 2600 8890 2604 8886 2612 8883 2619 8879 2627 8875
-	 2634 8871 2645 8868 2649 8864 2653 8860 2657 8860 2660 8856
-	 2664 8853 2668 8849 2672 8845 2675 8838 2679 8834 2679 8830
-	 2683 8826 2683 8823 2687 8815 2690 8811 2694 8808 2698 8804
-	 2702 8800 2705 8796 2709 8796 2709 8800 2713 8804 2717 8808
-	 2724 8811 2728 8815 2732 8815 2735 8815 2743 8815 2750 8815
-	 2758 8808 2765 8804 2773 8796 2777 8793 2780 8789 2784 8785
-	 2784 8781 2788 8778 2788 8774 2784 8770 2784 8766 2777 8755
-	 2773 8748 2769 8744 2769 8740 2765 8736 2769 8733 2769 8721
-	 2799 8710 2803 8710 2807 8706 2807 8703 2810 8699 2814 8695
-	 2818 8691 2822 8691 2825 8688 2833 8688 2840 8684 2848 8684
-	 2855 8680 2863 8680 2867 8680 2874 8680 2882 8680 2889 8680
-	 2897 8680 2908 8680 2915 8680 2927 8680 2934 8680 2942 8680
-	 2949 8680 2957 8680 2964 8680 2972 8680 2990 8680 2994 8680
-	 3002 8680 3005 8680 3013 8680 3020 8680 3028 8680 3035 8680
-	 3043 8680 3050 8676 3058 8676 3065 8676 3073 8673 3080 8673
-	 3088 8669 3092 8669 3103 8665 3118 8661 3125 8658 3133 8654
-	 3137 8650 3140 8646 3144 8643 3148 8639 3152 8635 3155 8635
-	 3159 8631 3163 8631 3167 8628 3174 8624 3185 8620 3197 8616
-	 3215 8609 3223 8605 3230 8601 3238 8601 3245 8598 3253 8594
-	 3260 8594 3268 8590 3272 8586 3279 8583 3287 8579 3294 8575
-	 3298 8571 3305 8571 3309 8568 3317 8564 3324 8560 3328 8556
-	 3332 8556 3335 8553 3339 8549 3343 8541 3343 8534 3343 8526
-	 3343 8519 3347 8511 3347 8504 3343 8496 3343 8493 3343 8485
-	 3343 8478 3343 8474 3343 8470 3343 8466 3343 8463 3347 8459
-	 3350 8455 3354 8451 3354 8448 3358 8444 3362 8444 3362 8440
-	 3362 8436 3365 8433 3365 8429 3365 8421 3365 8418 3362 8410
-	 3358 8406 3354 8403 3354 8399 3350 8395 3347 8391 3343 8391
-	 3339 8388 3335 8388 3332 8388 3332 8384 3328 8384 3332 8384
-	 3290 8388
+2 3 0 1 0 7 48 0 -1 0.000 0 0 7 0 0 1521
+	 1742 6936 1742 6929 1742 6925 1755 6935 1760 6935 1765 6935
+	 1775 6930 1780 6930 1785 6930 1790 6925 1795 6925 1800 6920
+	 1805 6915 1815 6910 1820 6905 1825 6905 1830 6905 1835 6905
+	 1840 6905 1845 6905 1850 6905 1860 6905 1870 6905 1875 6905
+	 1880 6910 1885 6910 1895 6910 1900 6915 1905 6915 1905 6920
+	 1910 6925 1910 6930 1910 6935 1910 6940 1905 6945 1900 6945
+	 1890 6950 1885 6955 1880 6960 1875 6960 1870 6965 1865 6970
+	 1865 6975 1865 6980 1860 6990 1860 6995 1865 7000 1865 7005
+	 1870 7010 1870 7015 1875 7015 1885 7015 1890 7015 1895 7015
+	 1900 7015 1905 7015 1915 7015 1920 7015 1930 7010 1935 7010
+	 1940 7010 1945 7010 1950 7010 1955 7010 1960 7015 1970 7015
+	 1970 7020 1975 7020 1980 7025 1985 7030 1990 7030 2000 7035
+	 2005 7035 2015 7040 2020 7040 2030 7040 2040 7045 2045 7045
+	 2050 7045 2055 7045 2065 7045 2070 7045 2075 7040 2085 7040
+	 2090 7040 2095 7040 2100 7040 2105 7040 2115 7040 2125 7040
+	 2130 7045 2135 7045 2140 7050 2145 7050 2150 7050 2155 7050
+	 2165 7055 2170 7060 2175 7060 2180 7065 2185 7070 2185 7075
+	 2185 7080 2185 7085 2185 7090 2185 7095 2185 7100 2185 7105
+	 2185 7110 2185 7115 2185 7120 2190 7125 2190 7130 2195 7135
+	 2195 7140 2200 7140 2205 7145 2205 7150 2215 7155 2220 7155
+	 2220 7160 2225 7160 2225 7165 2230 7170 2230 7175 2230 7180
+	 2230 7185 2230 7190 2230 7195 2230 7200 2230 7205 2230 7210
+	 2225 7215 2225 7220 2220 7225 2220 7230 2215 7235 2210 7240
+	 2205 7240 2205 7245 2200 7250 2195 7250 2195 7255 2190 7260
+	 2185 7260 2175 7265 2165 7265 2155 7265 2150 7265 2145 7265
+	 2140 7265 2135 7270 2130 7270 2125 7275 2120 7280 2115 7285
+	 2110 7290 2105 7295 2100 7295 2095 7300 2090 7300 2080 7300
+	 2075 7300 2065 7300 2060 7305 2055 7305 2050 7310 2050 7315
+	 2050 7320 2050 7325 2050 7330 2050 7335 2050 7340 2050 7345
+	 2050 7360 2050 7365 2050 7370 2050 7375 2045 7385 2045 7390
+	 2045 7395 2040 7405 2040 7410 2040 7415 2040 7420 2035 7430
+	 2035 7435 2035 7440 2035 7445 2035 7450 2030 7455 2030 7460
+	 2025 7470 2020 7480 2015 7490 2010 7495 2010 7500 2005 7505
+	 2005 7510 2005 7515 2000 7520 2000 7525 1995 7530 1990 7530
+	 1985 7535 1980 7535 1975 7535 1970 7540 1965 7540 1960 7545
+	 1955 7550 1955 7555 1950 7560 1950 7565 1950 7570 1950 7575
+	 1950 7580 1950 7585 1950 7590 1950 7595 1950 7600 1950 7605
+	 1950 7610 1950 7615 1950 7620 1945 7625 1940 7630 1935 7635
+	 1935 7640 1930 7640 1930 7645 1925 7645 1925 7650 1920 7650
+	 1915 7655 1910 7655 1905 7655 1900 7660 1895 7660 1890 7655
+	 1885 7655 1875 7650 1870 7650 1860 7645 1855 7645 1850 7645
+	 1845 7645 1840 7640 1835 7640 1830 7640 1825 7640 1815 7640
+	 1810 7640 1805 7640 1800 7640 1795 7640 1795 7645 1790 7650
+	 1790 7655 1790 7660 1810 7680 1815 7680 1815 7685 1820 7690
+	 1820 7695 1825 7695 1825 7700 1830 7705 1830 7710 1830 7715
+	 1830 7720 1835 7725 1835 7730 1835 7735 1835 7740 1835 7745
+	 1835 7750 1835 7755 1835 7760 1840 7765 1840 7770 1840 7775
+	 1845 7785 1845 7790 1845 7795 1850 7800 1850 7805 1850 7810
+	 1855 7815 1855 7820 1860 7830 1865 7835 1865 7840 1870 7845
+	 1870 7850 1870 7855 1870 7860 1865 7865 1860 7870 1855 7870
+	 1850 7875 1845 7880 1835 7880 1830 7890 1825 7895 1815 7900
+	 1810 7900 1805 7905 1800 7910 1795 7910 1795 7915 1790 7920
+	 1785 7925 1780 7930 1780 7935 1775 7940 1775 7945 1770 7950
+	 1765 7955 1765 7960 1760 7960 1755 7965 1755 7970 1750 7975
+	 1750 7980 1745 7985 1745 7990 1745 7995 1745 8000 1745 8005
+	 1745 8010 1745 8015 1745 8020 1750 8025 1750 8030 1755 8035
+	 1755 8040 1760 8045 1760 8050 1765 8055 1765 8060 1770 8060
+	 1775 8065 1785 8070 1790 8075 1795 8080 1800 8085 1800 8095
+	 1800 8100 1800 8105 1795 8105 1790 8110 1785 8115 1775 8115
+	 1765 8120 1755 8120 1750 8120 1745 8120 1740 8120 1735 8120
+	 1730 8125 1725 8125 1720 8130 1715 8135 1715 8140 1710 8140
+	 1705 8145 1700 8155 1690 8160 1685 8165 1675 8170 1670 8175
+	 1665 8180 1660 8185 1655 8190 1650 8190 1645 8200 1635 8205
+	 1635 8210 1630 8215 1630 8220 1630 8225 1625 8230 1625 8235
+	 1625 8240 1625 8245 1625 8250 1625 8260 1630 8270 1635 8280
+	 1640 8285 1640 8290 1635 8295 1635 8300 1630 8305 1625 8310
+	 1611 8343 1618 8343 1626 8343 1633 8343 1648 8346 1656 8346
+	 1663 8350 1671 8350 1674 8354 1682 8358 1686 8361 1693 8361
+	 1693 8365 1701 8369 1708 8376 1719 8380 1727 8388 1734 8391
+	 1738 8395 1745 8403 1753 8406 1757 8414 1760 8414 1760 8418
+	 1764 8425 1768 8429 1772 8433 1772 8440 1779 8448 1779 8451
+	 1783 8455 1783 8459 1787 8466 1787 8478 1790 8485 1790 8496
+	 1790 8504 1794 8511 1794 8519 1798 8526 1798 8530 1798 8534
+	 1798 8538 1798 8541 1798 8549 1798 8556 1798 8568 1798 8571
+	 1794 8575 1794 8583 1794 8590 1790 8594 1790 8601 1790 8609
+	 1790 8616 1790 8624 1798 8635 1798 8643 1798 8646 1802 8654
+	 1813 8669 1817 8669 1828 8676 1832 8676 1832 8684 1832 8691
+	 1835 8699 1839 8706 1843 8710 1847 8714 1847 8721 1850 8725
+	 1854 8733 1858 8736 1862 8744 1865 8748 1869 8751 1880 8755
+	 1888 8763 1895 8766 1903 8770 1910 8774 1914 8778 1918 8781
+	 1926 8785 1926 8789 1929 8789 1941 8793 1948 8796 1956 8796
+	 1967 8800 1974 8800 1986 8800 1989 8796 1997 8796 2023 8770
+	 2027 8766 2027 8763 2031 8759 2038 8751 2042 8748 2046 8744
+	 2049 8740 2053 8736 2061 8733 2061 8729 2068 8725 2072 8721
+	 2079 8721 2079 8718 2083 8718 2087 8714 2091 8714 2094 8714
+	 2098 8714 2102 8714 2106 8714 2113 8718 2121 8718 2128 8721
+	 2132 8721 2143 8721 2151 8721 2154 8721 2158 8721 2162 8721
+	 2166 8721 2173 8721 2181 8721 2188 8721 2192 8718 2199 8718
+	 2207 8714 2214 8714 2222 8706 2229 8706 2233 8703 2237 8699
+	 2241 8699 2244 8695 2248 8695 2252 8695 2259 8695 2263 8695
+	 2271 8695 2274 8695 2282 8699 2289 8703 2297 8706 2304 8706
+	 2308 8710 2315 8710 2319 8714 2327 8718 2334 8718 2342 8721
+	 2345 8721 2353 8721 2360 8725 2368 8729 2375 8729 2383 8733
+	 2390 8736 2398 8736 2405 8736 2413 8740 2424 8740 2435 8744
+	 2450 8748 2458 8748 2469 8751 2473 8751 2477 8751 2480 8751
+	 2488 8751 2495 8755 2503 8755 2510 8755 2522 8755 2529 8759
+	 2533 8759 2540 8759 2544 8763 2548 8766 2552 8766 2555 8766
+	 2559 8770 2567 8774 2574 8778 2582 8781 2585 8785 2593 8789
+	 2597 8793 2600 8796 2604 8800 2608 8804 2612 8811 2615 8811
+	 2623 8815 2627 8815 2634 8819 2638 8819 2642 8819 2645 8815
+	 2653 8811 2660 8808 2664 8800 2668 8796 2675 8796 2679 8793
+	 2683 8793 2687 8793 2690 8793 2698 8793 2705 8793 2717 8796
+	 2720 8796 2724 8800 2728 8808 2732 8811 2735 8815 2735 8819
+	 2743 8826 2750 8826 2758 8830 2769 8834 2777 8834 2788 8834
+	 2795 8834 2799 8830 2799 8826 2803 8823 2803 8819 2807 8811
+	 2810 8808 2822 8796 2829 8793 2840 8785 2848 8778 2855 8770
+	 2867 8763 2870 8755 2878 8748 2882 8740 2882 8733 2885 8725
+	 2889 8718 2889 8706 2893 8703 2893 8695 2897 8688 2900 8684
+	 2904 8680 2912 8676 2919 8676 2927 8676 2930 8676 2938 8676
+	 2949 8673 2953 8673 2964 8673 2972 8669 2983 8669 2994 8665
+	 3002 8665 3013 8665 3021 8661 3028 8661 3036 8661 3043 8661
+	 3051 8665 3058 8665 3066 8669 3073 8669 3081 8669 3088 8669
+	 3096 8665 3099 8665 3107 8661 3111 8661 3118 8658 3122 8658
+	 3126 8654 3129 8650 3133 8643 3137 8635 3137 8631 3137 8624
+	 3137 8620 3133 8613 3133 8609 3133 8601 3137 8594 3141 8586
+	 3144 8579 3148 8575 3156 8568 3171 8541 3171 8534 3174 8530
+	 3174 8526 3178 8519 3182 8515 3186 8511 3189 8508 3193 8504
+	 3197 8500 3201 8496 3204 8493 3208 8489 3216 8485 3223 8481
+	 3223 8478 3231 8474 3234 8470 3238 8466 3246 8463 3253 8455
+	 3261 8455 3268 8451 3276 8448 3279 8444 3287 8440 3294 8440
+	 3302 8436 3309 8436 3317 8433 3324 8433 3332 8429 3336 8425
+	 3343 8425 3351 8421 3358 8421 3366 8418 3369 8418 3373 8414
+	 3377 8410 3381 8410 3384 8406 3388 8406 3396 8403 3399 8399
+	 3403 8399 3407 8395 3411 8391 3414 8391 3418 8388 3418 8384
+	 3418 8380 3414 8373 3411 8369 3407 8365 3403 8361 3399 8358
+	 3396 8354 3392 8350 3384 8346 3384 8343 3381 8335 3373 8331
+	 3369 8316 3366 8313 3362 8305 3358 8301 3358 8294 3351 8286
+	 3347 8271 3343 8256 3343 8253 3339 8245 3336 8238 3336 8226
+	 3336 8219 3339 8178 3343 8170 3343 8163 3343 8159 3347 8159
+	 3347 8155 3343 8151 3343 8148 3339 8148 3336 8144 3332 8140
+	 3332 8136 3332 8129 3332 8125 3336 8121 3336 8118 3339 8114
+	 3343 8110 3347 8106 3351 8103 3354 8099 3362 8091 3366 8088
+	 3373 8084 3377 8076 3381 8073 3384 8069 3388 8069 3392 8065
+	 3396 8061 3403 8061 3407 8054 3414 8050 3422 8046 3433 8043
+	 3437 8039 3441 8035 3444 8031 3448 8031 3452 8028 3459 8024
+	 3463 8020 3467 8016 3470 8016 3474 8013 3478 8009 3482 8005
+	 3485 8001 3493 7998 3497 7994 3500 7990 3504 7986 3512 7983
+	 3519 7979 3527 7975 3534 7971 3545 7968 3549 7964 3553 7960
+	 3557 7960 3560 7956 3564 7953 3568 7949 3572 7945 3575 7938
+	 3579 7934 3579 7930 3583 7926 3583 7923 3587 7915 3590 7911
+	 3594 7908 3598 7904 3602 7900 3605 7896 3609 7896 3609 7900
+	 3613 7904 3617 7908 3624 7911 3628 7915 3632 7915 3635 7915
+	 3643 7915 3650 7915 3658 7908 3665 7904 3673 7896 3677 7893
+	 3680 7889 3684 7885 3684 7881 3688 7878 3688 7874 3684 7870
+	 3684 7866 3677 7855 3673 7848 3669 7844 3669 7840 3665 7836
+	 3669 7833 3669 7821 3699 7810 3703 7810 3707 7806 3707 7803
+	 3710 7799 3714 7795 3718 7791 3722 7791 3725 7788 3733 7788
+	 3740 7784 3748 7784 3755 7780 3763 7780 3767 7780 3774 7780
+	 3782 7780 3789 7780 3797 7780 3808 7780 3815 7780 3827 7780
+	 3834 7780 3842 7780 3849 7780 3857 7780 3864 7780 3872 7780
+	 3890 7780 3894 7780 3902 7780 3905 7780 3913 7780 3920 7780
+	 3928 7780 3935 7780 3943 7780 3950 7776 3958 7776 3965 7776
+	 3973 7773 3980 7773 3988 7769 3992 7769 4003 7765 4018 7761
+	 4025 7758 4033 7754 4037 7750 4040 7746 4044 7743 4048 7739
+	 4052 7735 4055 7735 4059 7731 4063 7731 4067 7728 4074 7724
+	 4085 7720 4097 7716 4115 7709 4123 7705 4130 7701 4138 7701
+	 4145 7698 4153 7694 4160 7694 4168 7690 4172 7686 4179 7683
+	 4187 7679 4194 7675 4198 7671 4205 7671 4209 7668 4217 7664
+	 4224 7660 4228 7656 4232 7656 4235 7653 4239 7649 4243 7641
+	 4243 7634 4243 7626 4243 7619 4247 7611 4247 7604 4243 7596
+	 4243 7593 4243 7585 4243 7578 4243 7574 4243 7570 4243 7566
+	 4243 7563 4247 7559 4250 7555 4254 7551 4254 7548 4258 7544
+	 4262 7544 4262 7540 4262 7536 4265 7533 4265 7529 4265 7521
+	 4265 7518 4262 7510 4258 7506 4254 7503 4254 7499 4250 7495
+	 4247 7491 4243 7491 4239 7488 4235 7488 4232 7488 4232 7484
+	 4228 7484 4232 7484 4190 7488 4187 7488 4179 7488 4175 7488
+	 4172 7488 4172 7491 4168 7491 4164 7491 4160 7491 4157 7495
+	 4153 7495 4138 7495 4127 7499 4119 7499 4112 7503 4104 7503
+	 4097 7503 4093 7503 4089 7503 4082 7499 4074 7495 4067 7491
+	 4067 7488 4059 7484 4055 7480 4044 7476 4037 7473 4029 7473
+	 4022 7469 4014 7469 4007 7469 3995 7469 3984 7469 3977 7469
+	 3969 7465 3969 7461 3965 7461 3965 7454 3962 7446 3962 7439
+	 3958 7431 3958 7424 3958 7416 3954 7413 3954 7409 3950 7409
+	 3947 7401 3947 7398 3947 7394 3943 7390 3935 7386 3932 7386
+	 3928 7383 3924 7379 3920 7375 3917 7375 3913 7371 3905 7368
+	 3898 7364 3879 7356 3872 7353 3868 7349 3860 7345 3857 7341
+	 3853 7338 3849 7334 3842 7330 3842 7326 3834 7323 3827 7319
+	 3819 7315 3812 7311 3804 7308 3797 7304 3793 7296 3789 7293
+	 3785 7289 3778 7285 3774 7281 3767 7278 3759 7274 3759 7278
+	 3752 7278 3744 7281 3733 7285 3725 7293 3722 7296 3718 7304
+	 3714 7308 3710 7311 3707 7311 3703 7308 3699 7304 3695 7304
+	 3692 7300 3688 7293 3684 7293 3677 7289 3669 7289 3662 7289
+	 3650 7289 3643 7285 3635 7285 3628 7281 3609 7278 3602 7278
+	 3590 7274 3590 7270 3583 7266 3579 7263 3572 7259 3568 7251
+	 3564 7248 3564 7240 3560 7240 3560 7236 3557 7233 3549 7225
+	 3542 7221 3538 7218 3534 7218 3527 7214 3515 7210 3512 7210
+	 3508 7210 3497 7206 3485 7206 3482 7206 3478 7206 3474 7203
+	 3474 7199 3474 7195 3474 7191 3470 7188 3470 7184 3467 7180
+	 3463 7173 3459 7169 3456 7169 3452 7165 3444 7165 3441 7165
+	 3437 7161 3429 7154 3426 7150 3422 7146 3418 7143 3411 7139
+	 3407 7135 3407 7131 3399 7128 3392 7124 3392 7120 3384 7120
+	 3377 7116 3369 7113 3366 7109 3362 7105 3354 7101 3351 7101
+	 3347 7098 3343 7098 3339 7098 3336 7098 3332 7098 3324 7098
+	 3321 7094 3324 7090 3324 7086 3328 7086 3328 7083 3332 7079
+	 3336 7075 3343 7071 3347 7064 3354 7060 3354 7056 3354 7053
+	 3354 7049 3351 7045 3351 7041 3343 7038 3339 7034 3332 7030
+	 3328 7030 3321 7030 3317 7026 3313 7026 3309 7026 3306 7023
+	 3302 7019 3298 7015 3294 7015 3291 7008 3291 7004 3291 7000
+	 3270 6990 3264 6985 3253 6981 3249 6981 3242 6978 3234 6974
+	 3227 6974 3219 6970 3212 6970 3204 6970 3201 6970 3197 6970
+	 3197 6966 3193 6966 3189 6966 3186 6966 3178 6966 3178 6963
+	 3174 6963 3167 6959 3163 6959 3159 6955 3156 6951 3152 6948
+	 3148 6944 3144 6940 3141 6936 3137 6929 3129 6925 3126 6921
+	 3118 6918 3111 6910 3103 6906 3099 6906 3092 6903 3084 6899
+	 3073 6899 3069 6899 3062 6899 3054 6899 3051 6903 3043 6903
+	 3032 6903 3024 6903 3021 6899 3013 6895 3006 6895 2998 6891
+	 2998 6888 2991 6884 2983 6884 2976 6880 2972 6876 2961 6873
+	 2961 6869 2957 6865 2957 6858 2957 6850 2957 6843 2957 6835
+	 2957 6831 2957 6828 2957 6820 2957 6816 2953 6813 2942 6801
+	 2934 6798 2927 6798 2919 6794 2908 6794 2900 6794 2889 6798
+	 2882 6798 2870 6798 2863 6801 2855 6801 2848 6801 2837 6801
+	 2825 6801 2810 6801 2799 6801 2788 6801 2777 6801 2765 6798
+	 2750 6794 2743 6794 2732 6786 2720 6783 2713 6779 2705 6775
+	 2702 6771 2694 6768 2683 6760 2672 6753 2664 6745 2653 6741
+	 2645 6734 2638 6730 2634 6730 2627 6726 2615 6726 2615 6723
+	 2608 6719 2604 6715 2600 6711 2593 6711 2593 6708 2585 6704
+	 2582 6700 2578 6696 2574 6693 2570 6689 2567 6685 2567 6678
+	 2563 6666 2563 6659 2563 6651 2563 6644 2559 6636 2555 6633
+	 2555 6629 2548 6621 2540 6621 2537 6621 2533 6618 2525 6614
+	 2514 6610 2507 6606 2503 6599 2495 6595 2488 6591 2480 6588
+	 2473 6584 2462 6576 2458 6569 2447 6565 2443 6565 2439 6565
+	 2435 6569 2432 6569 2428 6573 2428 6576 2424 6580 2424 6584
+	 2420 6591 2417 6595 2413 6599 2409 6606 2405 6610 2402 6610
+	 2394 6614 2387 6614 2379 6614 2372 6614 2360 6614 2349 6614
+	 2345 6610 2338 6610 2327 6606 2319 6603 2315 6599 2308 6595
+	 2304 6591 2297 6588 2289 6584 2282 6580 2274 6580 2267 6576
+	 2263 6576 2256 6573 2252 6569 2248 6565 2244 6561 2237 6558
+	 2233 6554 2233 6550 2226 6546 2222 6539 2218 6535 2211 6528
+	 2203 6520 2199 6516 2196 6509 2192 6509 2184 6501 2173 6501
+	 2169 6501 2166 6501 2162 6501 2158 6498 2154 6494 2151 6486
+	 2147 6483 2139 6479 2132 6475 2124 6471 2117 6468 2113 6468
+	 2106 6464 2098 6464 2091 6464 2087 6464 2072 6471 2064 6475
+	 2061 6479 2057 6483 2049 6486 2046 6494 2042 6501 2038 6505
+	 2034 6513 2034 6516 2031 6520 2031 6524 2027 6528 2023 6531
+	 2016 6531 2016 6535 2012 6539 2008 6539 2004 6539 1997 6543
+	 1993 6546 1989 6546 1986 6546 1978 6550 1971 6550 1963 6550
+	 1959 6546 1952 6546 1944 6543 1937 6539 1929 6535 1926 6531
+	 1918 6528 1910 6524 1907 6524 1899 6520 1892 6516 1884 6516
+	 1877 6516 1873 6516 1869 6513 1865 6513 1862 6513 1858 6516
+	 1854 6516 1847 6516 1843 6516 1839 6520 1835 6520 1828 6520
+	 1820 6520 1813 6520 1805 6520 1798 6520 1794 6520 1790 6524
+	 1787 6524 1783 6528 1783 6535 1779 6539 1779 6543 1779 6546
+	 1775 6550 1775 6554 1772 6554 1772 6558 1768 6558 1768 6561
+	 1768 6565 1768 6573 1768 6580 1768 6588 1772 6591 1772 6599
+	 1775 6603 1779 6610 1779 6614 1779 6621 1783 6629 1783 6636
+	 1783 6640 1783 6644 1783 6651 1779 6655 1779 6663 1783 6670
+	 1787 6681 1787 6685 1787 6689 1787 6693 1787 6696 1783 6700
+	 1779 6704 1775 6711 1779 6715 1787 6723 1794 6726 1802 6730
+	 1809 6730 1813 6734 1817 6738 1817 6741 1813 6745 1813 6749
+	 1813 6741 1809 6741 1805 6749 1802 6756 1794 6764 1790 6775
+	 1787 6779 1783 6786 1775 6801 1772 6809 1772 6813 1772 6820
+	 1768 6828 1768 6835 1768 6843 1768 6846 1764 6858 1764 6865
+	 1760 6869 1753 6880 1749 6891 1742 6895 1742 6903 1742 6906
+	 1742 6914 1742 6925 1742 6936
#! /bin/sh /usr/share/dpatch/dpatch-run
## 09_andre_images.dpatch by Andre Esser <aesser@netcologne.de>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Several computer images drawn by Andre Esser.

@DPATCH@
diff -urNad xfig~/Libraries/Computers/fujitsu-siemens.fig xfig/Libraries/Computers/fujitsu-siemens.fig
--- xfig~/Libraries/Computers/fujitsu-siemens.fig
+++ xfig/Libraries/Computers/fujitsu-siemens.fig
@@ -0,0 +1,60 @@
+#FIG 3.2  Produced by xfig version 3.2.5-alpha5
+Landscape
+Center
+Metric
+A4      
+100.00
+Single
+-2
+# Rack mounted PC
+# Drawn by Andre Esser
+1200 2
+0 33 #c7c3c7
+0 37 #e0e0e0
+0 48 #565151
+6 6930 3195 9990 3960
+6 9515 3316 9634 3422
+6 9515 3316 9634 3369
+1 3 0 1 0 4 50 0 25 0.000 1 0.0000 9541 3343 19 19 9541 3343 9555 3356
+1 3 0 1 0 4 50 0 25 0.000 1 0.0000 9607 3343 19 19 9607 3343 9621 3356
+-6
+6 9515 3369 9634 3422
+1 3 0 1 0 14 50 0 25 0.000 1 0.0000 9541 3395 19 19 9541 3395 9555 3409
+1 3 0 1 0 14 50 0 25 0.000 1 0.0000 9607 3395 19 19 9607 3395 9621 3409
+-6
+-6
+6 7247 3330 8212 3803
+6 7247 3330 8212 3503
+5 1 0 1 0 37 49 0 1 0.000 0 1 0 0 7721.500 2352.803 7247 3330 7737 3439 8196 3330
+2 2 0 1 0 37 50 0 6 0.000 0 0 7 0 0 5
+	 7247 3330 8212 3330 8212 3503 7247 3503 7247 3330
+-6
+6 7247 3630 8212 3803
+5 1 0 1 0 37 49 0 1 0.000 0 1 0 0 7721.500 2662.682 7247 3630 7737 3740 8196 3630
+2 2 0 1 0 37 50 0 6 0.000 0 0 7 0 0 5
+	 7247 3630 8212 3630 8212 3803 7247 3803 7247 3630
+-6
+-6
+6 8338 3330 9305 3803
+6 8338 3330 9305 3503
+5 1 0 1 0 37 49 0 1 0.000 0 1 0 0 8813.000 2350.697 8338 3330 8829 3439 9288 3330
+2 2 0 1 0 37 50 0 6 0.000 0 0 7 0 0 5
+	 8338 3330 9305 3330 9305 3503 8338 3503 8338 3330
+-6
+6 8338 3630 9305 3803
+5 1 0 1 0 37 49 0 1 0.000 0 1 0 0 8813.000 2660.595 8338 3630 8829 3740 9288 3630
+2 2 0 1 0 37 50 0 6 0.000 0 0 7 0 0 5
+	 8338 3630 9305 3630 9305 3803 8338 3803 8338 3630
+-6
+-6
+2 4 0 1 0 7 50 0 -1 0.000 0 0 2 0 0 5
+	 9660 3856 9660 3290 9502 3290 9502 3856 9660 3856
+2 4 0 1 0 48 50 0 9 0.000 0 0 2 0 0 5
+	 7194 3882 7194 3237 6983 3237 6983 3882 7194 3882
+2 4 0 1 0 48 50 0 9 0.000 0 0 2 0 0 5
+	 9937 3882 9937 3237 9726 3237 9726 3882 9937 3882
+2 4 0 1 0 48 50 0 27 0.000 0 0 2 0 0 5
+	 9634 3817 9634 3632 9528 3632 9528 3817 9634 3817
+2 2 0 1 0 33 53 0 15 0.000 0 0 7 0 0 5
+	 6930 3211 9977 3211 9977 3922 6930 3922 6930 3211
+-6
diff -urNad xfig~/Libraries/Computers/mirapoint.fig xfig/Libraries/Computers/mirapoint.fig
--- xfig~/Libraries/Computers/mirapoint.fig
+++ xfig/Libraries/Computers/mirapoint.fig
@@ -0,0 +1,85 @@
+#FIG 3.2
+Landscape
+Center
+Metric
+A4      
+100.00
+Single
+-2
+# Mirapoint M400
+# Drawn by Andre Esser
+1200 2
+0 32 #85807d
+0 33 #d2d2d2
+0 34 #3a3a3a
+0 35 #4573aa
+6 4455 2430 7541 4470
+6 4455 3617 7541 4470
+2 2 0 1 0 0 55 0 20 0.000 0 0 7 0 0 5
+	 4455 3617 7541 3617 7541 4470 4455 4470 4455 3617
+2 2 0 1 0 32 52 0 20 0.000 0 0 7 0 0 5
+	 4550 3664 4597 3664 4597 4423 4550 4423 4550 3664
+2 2 0 1 0 32 52 0 20 0.000 0 0 7 0 0 5
+	 4692 3664 4738 3664 4738 4423 4692 4423 4692 3664
+2 2 0 1 0 32 52 0 20 0.000 0 0 7 0 0 5
+	 4834 3664 4883 3664 4883 4423 4834 4423 4834 3664
+2 2 0 1 0 32 52 0 20 0.000 0 0 7 0 0 5
+	 4979 3664 5025 3664 5025 4423 4979 4423 4979 3664
+2 2 0 1 0 32 52 0 20 0.000 0 0 7 0 0 5
+	 5120 3664 5166 3664 5166 4423 5120 4423 5120 3664
+2 2 0 1 0 32 52 0 20 0.000 0 0 7 0 0 5
+	 5262 3664 5309 3664 5309 4423 5262 4423 5262 3664
+2 2 0 1 0 32 52 0 20 0.000 0 0 7 0 0 5
+	 5405 3664 5452 3664 5452 4423 5405 4423 5405 3664
+2 2 0 1 0 32 52 0 20 0.000 0 0 7 0 0 5
+	 5548 3664 5595 3664 5595 4423 5548 4423 5548 3664
+2 2 0 1 0 32 52 0 20 0.000 0 0 7 0 0 5
+	 5689 3664 5737 3664 5737 4423 5689 4423 5689 3664
+2 2 0 1 0 32 52 0 20 0.000 0 0 7 0 0 5
+	 5832 3664 5879 3664 5879 4423 5832 4423 5832 3664
+2 2 0 1 0 32 52 0 20 0.000 0 0 7 0 0 5
+	 5974 3664 6022 3664 6022 4423 5974 4423 5974 3664
+2 2 0 1 0 32 52 0 20 0.000 0 0 7 0 0 5
+	 6117 3664 6164 3664 6164 4423 6117 4423 6117 3664
+2 2 0 1 0 32 52 0 20 0.000 0 0 7 0 0 5
+	 6259 3664 6307 3664 6307 4423 6259 4423 6259 3664
+2 2 0 1 0 32 52 0 20 0.000 0 0 7 0 0 5
+	 6401 3664 6448 3664 6448 4423 6401 4423 6401 3664
+2 2 0 1 0 32 52 0 20 0.000 0 0 7 0 0 5
+	 6544 3664 6591 3664 6591 4423 6544 4423 6544 3664
+2 2 0 1 0 32 52 0 20 0.000 0 0 7 0 0 5
+	 6687 3664 6734 3664 6734 4423 6687 4423 6687 3664
+2 2 0 1 0 32 52 0 20 0.000 0 0 7 0 0 5
+	 6830 3664 6876 3664 6876 4423 6830 4423 6830 3664
+2 2 0 1 0 32 52 0 20 0.000 0 0 7 0 0 5
+	 6971 3664 7017 3664 7017 4423 6971 4423 6971 3664
+2 2 0 1 0 32 52 0 20 0.000 0 0 7 0 0 5
+	 7113 3664 7162 3664 7162 4423 7113 4423 7113 3664
+2 2 0 1 0 32 52 0 20 0.000 0 0 7 0 0 5
+	 7258 3664 7304 3664 7304 4423 7258 4423 7258 3664
+2 2 0 1 0 32 52 0 20 0.000 0 0 7 0 0 5
+	 7399 3664 7446 3664 7446 4423 7399 4423 7399 3664
+-6
+1 3 0 1 0 34 40 0 20 0.000 1 0.0000 5025 3189 47 47 5025 3189 5025 3236
+1 3 0 1 0 34 40 0 20 0.000 1 0.0000 5120 3094 47 47 5120 3094 5120 3142
+1 3 0 1 0 34 40 0 20 0.000 1 0.0000 5025 3000 47 47 5025 3000 5025 3047
+1 3 0 1 0 34 40 0 20 0.000 1 0.0000 4930 3094 47 47 4930 3094 4930 3142
+2 4 0 1 0 33 47 0 20 0.000 0 0 4 0 0 5
+	 5689 3379 4788 3379 4788 2621 5689 2621 5689 3379
+2 2 0 1 0 34 48 0 20 0.000 0 0 7 0 0 5
+	 4646 3330 7350 3330 7350 3379 4646 3379 4646 3330
+2 4 0 1 0 14 45 0 20 0.000 0 0 4 0 0 5
+	 5548 2905 4883 2905 4883 2713 5548 2713 5548 2905
+2 2 0 1 0 34 48 0 20 0.000 0 0 7 0 0 5
+	 4646 2621 7350 2621 7350 2667 4646 2667 4646 2621
+2 2 0 1 0 35 50 0 20 0.000 0 0 7 0 0 5
+	 4455 2430 7541 2430 7541 3617 4455 3617 4455 2430
+2 2 0 1 0 34 48 0 20 0.000 0 0 7 0 0 5
+	 4597 2762 7399 2762 7399 2809 4597 2809 4597 2762
+2 2 0 1 0 34 48 0 20 0.000 0 0 7 0 0 5
+	 4550 2905 7446 2905 7446 2951 4550 2951 4550 2905
+2 2 0 1 0 34 48 0 20 0.000 0 0 7 0 0 5
+	 4597 3189 7399 3189 7399 3236 4597 3236 4597 3189
+2 2 0 1 0 34 48 0 20 0.000 0 0 7 0 0 5
+	 4550 3047 7446 3047 7446 3094 4550 3094 4550 3047
+-6
diff -urNad xfig~/Libraries/Computers/sun_d1000.fig xfig/Libraries/Computers/sun_d1000.fig
--- xfig~/Libraries/Computers/sun_d1000.fig
+++ xfig/Libraries/Computers/sun_d1000.fig
@@ -0,0 +1,21 @@
+#FIG 3.2
+Landscape
+Center
+Metric
+A4      
+100.00
+Single
+-2
+# SUN D1000 disk shelf
+# Drawn by Andre Esser
+1200 2
+0 33 #7b79a5
+6 2160 4230 5400 5490
+2 2 0 1 0 33 50 0 50 0.000 0 0 7 0 0 5
+	 5355 5223 2205 5223 2205 5466 5355 5466 5355 5223
+2 2 0 1 0 33 49 0 40 0.000 0 0 7 0 0 5
+	 2302 5247 2641 5247 2641 5441 2302 5441 2302 5247
+2 2 0 1 0 33 49 0 9 0.000 0 0 7 0 0 5
+	 2205 4254 5355 4254 5355 5223 2205 5223 2205 4254
+4 0 0 49 0 0 7 0.0000 4 73 153 2399 5393 Sun\001
+-6
diff -urNad xfig~/Libraries/Computers/sunfire_v120.fig xfig/Libraries/Computers/sunfire_v120.fig
--- xfig~/Libraries/Computers/sunfire_v120.fig
+++ xfig/Libraries/Computers/sunfire_v120.fig
@@ -0,0 +1,33 @@
+#FIG 3.2
+Landscape
+Center
+Metric
+A4      
+73.00
+Single
+-2
+# SunFire V120
+# Drawn by Andre Esser
+1200 2
+0 49 #444444
+0 80 #73758c
+# SunfireV120
+6 818 3167 3841 3435
+2 1 0 1 0 7 30 0 20 0.000 0 0 -1 0 0 5
+	 2161 3247 2108 3300 2161 3355 2216 3300 2161 3247
+2 2 0 1 0 80 45 0 20 0.000 0 0 -1 0 0 5
+	 3840 3435 818 3435 818 3167 3840 3167 3840 3435
+2 2 0 1 0 31 40 0 20 0.000 0 0 -1 0 0 5
+	 3570 3195 3732 3195 3732 3221 3570 3221 3570 3195
+2 2 0 1 0 0 25 0 19 0.000 0 0 -1 0 0 5
+	 3544 3273 3786 3273 3786 3329 3544 3329 3544 3273
+2 2 0 1 0 14 35 0 20 0.000 0 0 -1 0 0 5
+	 854 3173 882 3173 882 3240 854 3240 854 3173
+3 2 0 1 0 49 45 0 25 0.000 0 0 0 3
+	 818 3249 2036 3303 818 3356
+	 0.000 -1.000 0.000
+3 2 0 1 0 49 45 0 25 0.000 0 0 0 3
+	 3841 3249 2623 3303 3841 3356
+	 0.000 -1.000 0.000
+4 0 0 30 0 27 7 0.0000 4 90 270 2270 3329 Sun\001
+-6
diff -urNad xfig~/Libraries/Computers/sunfire_v240.fig xfig/Libraries/Computers/sunfire_v240.fig
--- xfig~/Libraries/Computers/sunfire_v240.fig
+++ xfig/Libraries/Computers/sunfire_v240.fig
@@ -0,0 +1,46 @@
+#FIG 3.2  Produced by xfig version 3.2.5-alpha5
+Landscape
+Center
+Metric
+A4      
+100.00
+Single
+-2
+# SunFire V240
+# Drawn by Andre Esser
+1200 2
+0 80 #73758c
+6 6210 3544 6688 3756
+1 2 0 1 0 7 44 -1 -1 0.000 1 0.0000 6449 3650 239 106 6210 3650 6688 3650
+2 1 0 1 0 7 30 0 20 0.000 0 0 -1 0 0 5
+	 6325 3598 6280 3651 6325 3706 6372 3651 6325 3598
+4 0 0 30 0 27 6 0.0000 4 90 192 6418 3680 Sun\001
+-6
+2 2 0 1 0 31 40 0 20 0.000 0 0 -1 0 0 5
+	 7710 3510 7872 3510 7872 3536 7710 3536 7710 3510
+2 2 0 1 0 14 35 0 20 0.000 0 0 -1 0 0 5
+	 4994 3488 5022 3488 5022 3555 4994 3555 4994 3488
+2 1 0 1 0 80 44 -1 5 0.000 0 0 7 0 0 10
+	 7652 3512 5220 3510 5221 3534 6210 3580 6305 3533 6447 3522
+	 6597 3538 6680 3581 7650 3532 7641 3504
+2 2 0 1 0 80 45 0 20 0.000 0 0 -1 0 0 5
+	 7980 4050 4950 4050 4950 3482 7980 3482 7980 4050
+2 1 0 1 0 80 44 -1 5 0.000 0 0 -1 0 0 12
+	 4987 3778 6194 3718 6314 3777 6456 3793 6616 3769 6684 3733
+	 7919 3769 7920 3816 6668 3870 6222 3866 4996 3823 4999 3782
+2 1 0 1 0 80 44 -1 5 0.000 0 0 -1 0 0 4
+	 5661 4051 6178 3978 6755 3975 7257 4040
+2 2 0 1 0 80 40 -1 20 0.000 0 0 -1 0 0 5
+	 5014 3625 5255 3625 5255 3693 5014 3693 5014 3625
+3 2 0 1 0 80 44 0 5 0.000 0 0 0 3
+	 4989 3589 6165 3645 4989 3739
+	 0.000 -1.000 0.000
+3 2 0 1 0 80 44 0 5 0.000 0 0 0 3
+	 4992 3855 6165 3915 4992 4005
+	 0.000 -1.000 0.000
+3 2 0 1 0 80 44 0 5 0.000 0 0 0 3
+	 7922 3860 6750 3915 7924 4005
+	 0.000 -1.000 0.000
+3 2 0 1 0 80 44 0 5 0.000 0 0 0 3
+	 7922 3591 6750 3645 7921 3737
+	 0.000 -1.000 0.000
diff -urNad xfig~/Libraries/Computers/sunfire_v440.fig xfig/Libraries/Computers/sunfire_v440.fig
--- xfig~/Libraries/Computers/sunfire_v440.fig
+++ xfig/Libraries/Computers/sunfire_v440.fig
@@ -0,0 +1,79 @@
+#FIG 3.2
+Landscape
+Center
+Metric
+A4      
+100.00
+Single
+-2
+# SunFire V440
+# Drawn by Andre Esser
+1200 2
+0 46 #444444
+0 74 #f7f7f7
+0 76 #414541
+0 82 #635dce
+6 1980 3869 4971 5310
+1 1 0 1 0 7 50 0 -1 0.000 1 0.0000 3331 4103 181 126 3331 4103 3511 4229
+2 3 0 1 0 82 48 0 20 0.000 0 0 -1 0 0 5
+	 2791 5094 1980 5058 1980 5040 2791 5076 2791 5094
+2 3 0 1 0 82 48 0 20 0.000 0 0 -1 0 0 5
+	 2791 4878 1980 4914 1980 4932 2791 4896 2791 4878
+2 3 0 1 0 82 48 0 20 0.000 0 0 -1 0 0 5
+	 2791 4445 1980 4481 1980 4499 2791 4463 2791 4445
+2 3 0 1 0 82 48 0 20 0.000 0 0 -1 0 0 5
+	 2791 4265 1980 4229 1980 4211 2791 4247 2791 4265
+2 3 0 1 0 82 48 0 20 0.000 0 0 -1 0 0 5
+	 2791 4031 1980 4067 1980 4085 2791 4050 2791 4031
+2 2 0 1 0 76 55 0 21 0.000 0 0 -1 0 0 5
+	 2809 3927 2034 3927 2034 5274 2809 5274 2809 3927
+2 3 0 1 0 82 48 0 20 0.000 0 0 -1 0 0 5
+	 2809 4680 1980 4644 1980 4626 2791 4662 2809 4680
+2 2 0 1 0 82 60 0 20 0.000 0 0 -1 0 0 5
+	 2809 5310 1980 5310 1980 3869 2809 3869 2809 5310
+2 2 0 1 0 82 60 0 20 0.000 0 0 -1 0 0 5
+	 2809 3869 4971 3869 4971 5310 2809 5310 2809 3869
+2 3 0 1 0 46 50 0 22 0.000 0 0 -1 0 0 7
+	 4899 5004 3511 4932 2899 4932 2899 5094 3511 5094 4899 5040
+	 4899 5004
+2 3 0 1 0 46 50 0 22 0.000 0 0 -1 0 0 7
+	 4899 4716 3511 4644 2899 4644 2899 4805 3511 4805 4899 4752
+	 4899 4716
+2 2 0 1 0 74 50 0 20 0.000 0 0 -1 0 0 5
+	 2664 4662 3061 4662 3061 4787 2664 4787 2664 4662
+2 3 0 1 0 46 50 0 22 0.000 0 0 -1 0 0 6
+	 4124 5274 3511 5220 3079 5220 2899 5274 4106 5274 4124 5274
+2 2 0 1 0 7 50 0 -1 0.000 0 0 -1 0 0 5
+	 2809 3923 4899 3923 4899 5274 2809 5274 2809 3923
+2 2 0 1 0 74 50 0 20 0.000 0 0 -1 0 0 5
+	 3710 4283 4899 4283 4899 4410 3710 4410 3710 4283
+2 2 0 1 0 46 50 0 22 0.000 0 0 -1 0 0 5
+	 2899 4265 3565 4265 3565 4463 2899 4463 2899 4265
+2 3 0 1 0 46 50 0 22 0.000 0 0 -1 0 0 11
+	 2917 3959 3151 4013 3213 3973 3283 3953 3373 3953 3463 3983
+	 3513 4018 3547 4013 4719 3923 2917 3923 2917 3959
+2 2 0 1 0 46 50 0 22 0.000 0 0 -1 0 0 5
+	 2827 3887 2863 3887 2863 5310 2827 5310 2827 3887
+3 2 0 1 0 46 48 0 22 0.000 0 0 0 3
+	 4899 4499 3547 4590 4899 4680
+	 0.000 -1.000 0.000
+3 2 0 1 0 46 48 0 22 0.000 0 0 0 3
+	 4899 5076 3547 5166 4899 5256
+	 0.000 -1.000 0.000
+3 2 0 1 0 46 48 0 22 0.000 0 0 0 3
+	 4899 4787 3547 4878 4899 4968
+	 0.000 -1.000 0.000
+3 2 0 1 0 46 50 0 22 0.000 0 0 0 4
+	 2899 4499 3097 4553 2899 4626 2899 4608
+	 0.000 -1.000 -1.000 0.000
+3 2 0 1 0 46 48 0 22 0.000 0 0 0 3
+	 4899 3977 3547 4067 4899 4157
+	 0.000 -1.000 0.000
+3 2 0 1 0 46 50 0 22 0.000 0 0 0 4
+	 2899 4013 3097 4067 2899 4139 2899 4121
+	 0.000 -1.000 -1.000 0.000
+3 2 0 1 0 46 50 0 22 0.000 0 0 0 4
+	 2899 5112 3097 5166 2899 5238 2899 5220
+	 0.000 -1.000 -1.000 0.000
+4 0 0 50 0 0 6 0.0000 4 78 174 3241 4139 Sun\001
+-6
